src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -756,9 +720,8 @@ public static BigInteger DivRem(BigInteger dividend, BigInteger divisor, out Big
 
             if (trivialDividend && trivialDivisor)
             {
-                BigInteger quotient;
-                (quotient, remainder) = Math.DivRem(dividend._sign, divisor._sign);
-                return quotient;
+                remainder = dividend._sign % divisor._sign;
```

Comment:
This was using `DivRem` as an optimization, it should continue doing so.

Reply:
Fixed